test/beast/core/async_base.cpp

```diff
@@ -635,7 +635,7 @@ class async_base_test : public beast::unit_test::suite
     }
 
     // Asynchronously send a message multiple times, once per second
-    template <class AsyncWriteStream, class T, class WriteHandler>
+    template <class AsyncWriteStream, class T, BOOST_BEAST_ASYNC_TPARAM2 WriteHandler>
```

Comment:
This code should be copied from the javadoc containing it in the file async_base.hpp